.github/workflows/ci.yml

```diff
@@ -67,6 +68,7 @@ jobs:
         run: pnpm run test-build -- --runInBand
 
   lint:
+    timeout-minutes: 10
```

Comment:
Could you send this in a separate PR @dominikg?

Reply:
done https://github.com/vitejs/vite/pull/6843